src/LinuxTracing/LinuxTracingUtils.cpp

```diff
@@ -153,8 +142,8 @@ std::vector<int> ParseCpusetCpus(const std::string& cpuset_cpus_content) {
 // An empty result indicates an error, as trying to start a process with an
 // empty cpuset fails with message "cgroup change of group failed".
 std::vector<int> GetCpusetCpus(pid_t pid) {
-  std::optional<std::string> cgroup_content_opt = ReadCgroupContent(pid);
-  if (!cgroup_content_opt.has_value()) {
+  ErrorMessageOr<std::string> cgroup_content_opt = ReadCgroupContent(pid);
```

Comment:
The `_opt` suffix was referring to `std::optional`, I guess it should be changed (or removed).

Reply:
done.